sips/minutes/2016-10-25-sip-minutes.md

```diff
@@ -0,0 +1,173 @@
+that it wouldn't be good to contradict last month's vote on the proposal.
+Heather says that she's in favor of trailing commas because makes beginners be
+less confused by the syntax and help people get started with the language.
+Iulian doesn't like the specialized version, he would prefer a general version.
+
+**Outcome**: All the Committee decides to vote on accepting the proposal again for a final
+review with a clear specification of the changes. Seth and Adriaan vote against,
+the rest of the Committee except Josh vote in favor of it. According to our new
+rules, Josh gets to decide whether the proposal will be readmitted for review or
+cancelled (**NOTE**: Josh votes in favor of trailing commas one week after the
+meeting). Trailing commas will be reviewed in November.
+
+### Discussion of SIP-20: Improved Lazy Val Initialization
+
+Sébastien comments the changes that Dmitry has introduced to the proposal.
+Several new schemes have been added to the proposal based on his experience in
+Dotty, in which the championed scheme is already implemented.
+
+There are no clear winner in the case of local lazy vals, but there seems to be
+two clear candidates for the non-local lazy vals (B4-general and B6).
+Both are faster than the existing implementation in the contended case but B4
+general is 4x and B6 is only 2x. However, for the uncontended case B4 general is
+30% slower than the existing implementation, while B6 is on par, up to +-5%.
+Dmitry recommends the B6 case because it's a *pure* win. Memory-wise, B6 would
+have a smaller memory footprint.
+
+Sébastien comments on the increase of the bytecode for the getter of the lazy
+val, which is 4x bigger. This could make a difference, although the benchmarks
+do not show any negative impact of it (probably because the previous
+implementation was bigger than the limits set by the JVM to benefit from
+code inlining).
+
+Jorge comments that there's no implementation of this proposal for scalac. This
+means that someone should step up and provide such implementation, because
+Dmitry does not have time for it. Adriaan comments on the fact that B6 uses
+`sun.misc.Unsafe`, which he prefers that the compiler doesn't depend on to run
+in other platforms that don't allow it (Google App Engine). Sébastien highlights
+that the proposal could use var handles, which are planned to be shipped on Java
+9.
+
+Heather proposes to mark this proposal as dormant, since it's lacking an
+implementation. She wants to let other people claim its ownership and provide an
+implementation if they care about this issue.
+
+**Outcome**: The Committee agrees to mark the proposal as dormant and let
+someone pick it up and provide an implementation. Ideally, this implementation
+should run in Java 8 and don't depend on var handles. The SIP Process Lead will
+wait some time to receive another candidate from the current author, Dmitry. If
+he doesn't propose anything, the SIP will be marked as dormant.
```

Comment:
Yeah, it's contradictory because we changed our mind several times during the discussion. The one at the end is the final decision. I would update this part of the document to make it clearer.